src/types/observablemap.ts

```diff
@@ -40,9 +41,19 @@ export interface IMap<K, V> {
     get(key: K): V | undefined
     has(key: K): boolean
     set(key: K, value?: V): this
+    // prettier-ignore
+    [Symbol.toStringTag]: "Map"; // necessary semicolon
```

Comment:
prettier gets rid of this semi, but it is needed. Not familiar enough with prettier to know if there's a better way.

(Or maybe there's a better way to let TS know it's the end of the statement, but I don't know it).